### ISSUE TITLE
[Bug]: Arrow in main 

### ISSUE THREAD

**Issue by keilmillerjr** [bottlesdevs/Bottles#1795]
### Describe the bug

bottles 2022.6.14-9
http://aur.archlinux.org/packages/bottles

Clicking on a bottle yields this error:

```
Traceback (most recent call last):
  File "/usr/share/bottles/bottles/views/list.py", line 159, in show_details
    self.window.show_details_view(config=config)
  File "/usr/share/bottles/bottles/window.py", line 339, in show_details_view
    self.page_details.set_config(config)
  File "/usr/share/bottles/bottles/views/details.py", line 184, in set_config
    self.view_preferences.set_config(config=config)
  File "/usr/share/bottles/bottles/views/bottle_preferences.py", line 336, in set_config
    self.switch_runtime.handler_block_by_func(self.__toggle_runtime)
TypeError: nothing connected to <bound method PreferencesView.__toggle_runtime of <bottle_preferences.PreferencesView object at 0x7fa7a1337a00 (DetailsPreferences at 0x55a189faf0d0)>>
```

### To Reproduce

1. Click on a bottle.

### Package

AUR ("bottles" package)

### Distribution

Kernel: 5.18.12-arch1-1 arch: x86_64 bits: 64

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :1
    Wayland: false
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/share/vulkan/icd.d/radeon_icd.x86_64.json:/usr/share/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.18.12-arch1-1
Distro:
    Name: Arch Linux
    Version: Unknown
Disk:
    Total: 983046811648
    Free: 704720101376
RAM:
    MemTotal: 15.5GiB
    MemAvailable: 11.5GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.35'
Bottles_envs: null
```

**Comment by axtloss** [comment]
The bottles aur package is outdated, try using the package from flathub and check if the issue persists.

**Comment by mirkobrombin** [comment]
Fixed in 2022.6.28.

**Comment by keilmillerjr** [comment]
A closer inspection reveals lies in the pkgbuild.

```
pkgver=2022.6.14
source=("${pkgname}-2022.5.28-trento-3.tar.gz::https://github.com/bottlesdevs/Bottles/archive/2022.5.28-trento-3.tar.gz")
```

The flathub version works correctly. Thank you.
